package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This project contains all the server code for the rif relay system.",
   "main": "dist/index.js",
   "scripts": {
-    "build": "scripts/build",
+    "build": "scripts/dist",
```

Comment:
I know it's the same, but I think it's more common to have the `build` script, instead of `dist`. Could we rename `dist` to `build`?

Reply:
Done